lib/logstash/outputs/kafka.rb

```diff
@@ -268,6 +268,9 @@ def retrying_send(batch)
       futures.each_with_index do |future, i|
         begin
           result = future.get()
+        rescue org.apache.kafka.common.errors.RecordTooLargeException => x
```

Comment:
@robbavey there's another PR that catches this exception during  `producer.send`
 https://github.com/logstash-plugins/logstash-output-kafka/pull/191/files#diff-c6a637e53249d2c5a0ee842b0bc19e82R265 instead of this PR's `future.get`

I believe this one makes sense as the other operation will only return the future so no exception should be raised, right?

Reply:
Yes, this PR ups the visibility of an error that was already being logged at `debug` in the following `rescue => e` block. I'm kind of tempted to change this PR to just increase the log level of the blanket `rescue` to increase the visibility of _any_ error caught here, and reduce if the noise levels get too high.

Reply:
Agreed, we want to know whenever an error happens, otherwise logstash will either be silently retrying things or dropping data (Both terrible).